_opensearch/rest-api/analyze-apis/terminology.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Analysis API Terminology
+parent: Analyze APIs
+grand_parent: REST API reference
+nav_order: 1
+---
+
+# Terminology
+
+The following sections provide descriptions of important text analysis terms. 
+
+## Analyzers
+
+Analyzers instruct OpenSearch how to index and search text. Analyzers comprise three components: a tokenizer, zero or more token filters, and zero or more character filters. 
```

Comment:
Suggestion only:
Grammatically, "Analyzers comprise three components..." I think is correct. But it sounds stiff and stops me. Does it sound awkward to skip to singular here?
"An analyzer comprises three components..." or even "An analyzer is composed of three components..."?
Even "Analyzers are composed of three components..." sounds better to me.